src/ocl/convolutionocl.cpp

```diff
@@ -150,26 +377,61 @@ std::vector<Solution> FindConvolution(const ExecutionContext& ctx,
     auto sol             = boost::optional<miopenConvSolution_t>{};
     const auto& conv     = problem.GetConv();
     const auto& findMode = conv.findMode;
+    auto fallback        = FallbackPath();
+    std::vector<miopenConvSolution_t> sols;
+    std::vector<miopenConvSolution_t> ufdb_sols;
 
     if(findMode.IsFast(ctx) || findMode.IsHybrid(ctx))
     {
-        auto fallback = bool{};
-        auto sols     = conv.GetSolutions(ctx, problem, 1, &fallback, &invoke_ctx);
+        if(findMode.IsTrustVerify(ctx))
+        {
+            ufdb_sols = miopen::GetSolutions<UserFindDb>(ctx, problem, 1, &invoke_ctx);
+            if(!ufdb_sols.empty())
+                sols = ufdb_sols;
+            else
+                sols = conv.GetSolutions(ctx, problem, 2, &fallback, &invoke_ctx);
+        }
+        else
+            sols = conv.GetSolutions(ctx, problem, 1, &fallback, &invoke_ctx);
```

Comment:
Returning more than 1 solution from GetSolutions can cause warning message for workspace size in hybrid dynamic mode. Separated paths here.